### PR TITLE
Single page

### PR DESCRIPTION
**Pull request by mutinhiri**
In this PR i added changes for full single page app using react routing
- The website consist of 3 pages Home, Calculator, Quote

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Maths Magician project app
+# Maths Magician project App
```
Comment (rloterh) [inline README.md:3]: Your _ReadMe_ needs some improvements so as to make it appear more professional. Kindly note that _ReadMe_ is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed and professional. Also, it needs to be done well with respect to this capstone project,  which will likely feature on your portfolio. Therefore, kindly ensure to improve upon these aspects of your ReadMe:
- [ ] _Project description_, _how to set up_, _run and test this project_ **should be detailed, clear, and well** organized.
- [ ] The _code_ aspects in each procedure should be quoted with **backticks**.
- [ ] Include necessary screenshots, project demo links where applicable so as to give a birds-eye view of your project
- [ ] All _typos_ **should be corrected**. You can crosscheck by installing the [Grammarly plugin](https://www.grammarly.com/)

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,71 @@
...
 
 const App = () => (
-  <div className="App">
-    <Calculator />
-  </div>
+  <Router>
+    <header className="header">
+      <h1>Math Magician</h1>
+      <ul className="navigation">
+        <li><a href="home">Home</a></li>
```
Comment (rloterh) [inline src/App.js]: - [x] Your app does not navigate to the _home page_ when the app is launched. As a best practice, kindly ensure that your app navigates to the _home page_ by default, on port `http://localhost:3000/`.

**localhost:3000:**
image

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,71 @@
...
+        <li><a href="calculator">Calculator</a></li>
+        <li><a href="quote">Quote</a></li>
```
Comment (rloterh) [inline src/App.js]: - [ ] Your app **refreshes** when a menu link of a page is clicked. Kindly ensure that your app conforms to the SPA feature. You can improve upon this aspect by replacing the `<a>` tags with react `Link` tags.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Bunbee, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._
<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @mutinhiri, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**